packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -61,7 +61,9 @@ class ServiceConnectionManager {
       );
   }
 
-  late final ServiceManager<VmServiceWrapper> serviceManager;
+  late final ServiceManager<VmServiceWrapper> _serviceManager;
+
+  ServiceManager<VmServiceWrapper> get serviceManager => _serviceManager;
```

Comment:
nit: order getters before private fields https://github.com/flutter/devtools/blob/master/STYLE.md#order-of-getters-and-setters